src/GraphMetadataViewExtension/GraphMetadataViewModel.cs

```diff
@@ -99,6 +103,8 @@ public BitmapImage Thumbnail
         /// </summary>
         public ObservableCollection<CustomPropertyControl> CustomProperties { get; set; }
 
+        public LinterExtensionDescriptor CurrentLinter => (viewLoadedParams.DynamoWindow.DataContext as DynamoViewModel).Model.LinterManager.ActiveLinter;
```

Comment:
Is there any other option to access the DynamoModel?  I see that this is being used a lot (in other cases) but I wasn't sure is there was another recommendation @mjkkirschner